lib/invoker.js

```diff
@@ -98,20 +98,25 @@ module.exports = async (userFunctionUri, options, shutdownSignal) => {
 
     const initHookTimeout = options.$initTimeoutMs || 10000;
     return executeInit(userFunction.$init, initHookTimeout, server)
+        .then(loadRiffService)
```

Comment:
`executeInit` and `loadRiffService` are independent async tasks. We could run them in parallel to reduce startup latency.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all would work perfectly